app/enterprise/2.3.x/plugins/oidc-curity.md

```diff
@@ -0,0 +1,209 @@
+
+More information is available in the [Code Flow][curity-code-flow-tutorial] tutorial.
+
+### Configure OpenID Connect in the Kong Developer Portal
+
+Enabling the Kong Developer Portal is outlined in the [Kong Dev Portal Documentation][kong-dev-portal-doc] and not covered in this article. The documentation also covers how to configure the [OpenID Connect Plugin][kong-dev-portal-doc-oidc].
+
+#### Example Configuration Object
+
+Below is an example configuration object that is used to configure the OIDC plugin for the Dev Portal.
+
+```json
+{
+    "redirect_uri": ["https://kong-dev-portal:8004/default/auth"],
+    "consumer_by": ["username","custom_id","id"],
+    "leeway": 1000,
+    "scopes": ["openid","profile","email"],
+    "logout_query_arg": "logout",
+    "login_redirect_uri": ["https://kong-dev-portal:8003"],
+    "login_action": "redirect",
+    "logout_redirect_uri": ["https://kong-dev-portal:8003"],
+    "ssl_verify": false,
+    "client_id": ["kong-dev-portal-client"],
+    "forbidden_redirect_uri": ["https://kong-dev-portal:8003/unauthorized"],
+    "client_secret": ["Pa$$w0rd!"],
+    "issuer": "https://idsvr.example.com/oauth/v2/oauth-anonymous/",
+    "logout_methods": ["GET"],
+    "consumer_claim": ["email"],
+    "login_redirect_mode": "query"
+}
+```
+
+![Enable OIDC in Kong Dev Portal](/assets/images/docs/dev-portal/curity/kong-dev-portal.png)
+
+### Curity Authentication Action
+
+An Authentication Action to automatically provision the user to the Kong Developer Portal is available in the Curity GitHub repository. Using the Action is not mandatory as the user could be provisioned in other ways, such as manually through the Kong Developer portal login screen. However, using the Authentication Action would streamline the user flow as the Action will take the user's full name and the email from the Curity Authenticator and automatically provision that to the Kong Developer Portal using the exposed API.
+
+The [The Kong Developer Portal User Provisioner][curity-kong-dev-portal-user-provisioner] action is available as open-source and can be forked to fit the needs of the environment as needed.
+
+#### Configuration
+
+This Action is straightforward to configure. An HTTP Client is needed to communicate with the Kong Dev Portal API. By default, the **HTTP Authentication** can be left out. Only a correct scheme needs to be configured (HTTP or HTTPS).
+
+The Action also configures the URL to the registration endpoint of the Kong Developer Portal. Here the scheme needs to match what's configured in the HTTP Client used.
+
+![Kong Dev Portal User Provisioner](/assets/images/docs/dev-portal/curity/kong-dev-portal-action.png)
+
+When the action is created, it can be assigned to the Authenticator(s) used in the client configured in the Curity Identity Server as described above.
```

Comment:
```suggestion
When the action is created, it can be assigned to the Authenticators used in the client configured in the Curity Identity Server as described above.
```